crates/api_models/src/payments.rs

```diff
@@ -63,9 +63,16 @@ pub struct PaymentsRequest {
     #[serde(default, deserialize_with = "amount::deserialize_option")]
     pub amount: Option<Amount>,
 
+    #[schema(value_type = Option<RoutingAlgorithm>, example = json!({
+        "type": "single",
+        "data": "stripe"
+    }))]
+    pub routing: Option<serde_json::Value>,
```

Comment:
If possible, have this as a concrete type which will enforce type saftey so that we don't accept invalid data. While inserting it into the database it can be made into `serde_json::value`

Reply:
The type will have different interpretations for OSS and VAS, that's why the decision was made to make it generic.

Reply:
As per discussion with @jarnura, we will need to come up with some other strategy. For now we will go with this.